subsys/bluetooth/crypto_toolbox/Kconfig

```
@@ -0,0 +1,18 @@
+config BT_CRYPTO_TOOLBOX
+    bool "Cryptographic Toolbox"
```

I think this should not have a prompt. I'm my mind this is not a user option.
```suggestion
    bool
```